### PR TITLE
feat:added cart page

### PR DESCRIPTION
**Pull request by GuptaEesh**
>Added cart page
Added cart route
Added cart path for barrel import

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
Good use of barrel imports and Clean well-structured code
All good practices followed
Good to merge

File: src/Pages/cart/cart.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    let totalPrice = items.reduce(
+        (total, item) => (total += Number(item.price * item.quantity)),
+        0
+    )
+    let discount = Math.round(0.05 * totalPrice)
+    let delivery = totalPrice > 200 ? 0 : Math.round(0.19 * totalPrice)
+    return items.length !== 0 ? (
+        <div
+            className="flex flex-wrap flex-row justify-space-around"
+            style={{ paddingTop: '5rem' }}
```
Comment (adith-a-danthi) [inline src/Pages/cart/cart.js:20]: Can use a utility class for this

File: src/Pages/cart/cart.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        >
+            <section>
+                <h1 className="bold size-16" style={{ margin: '1rem' }}>
```
Comment (adith-a-danthi) [inline src/Pages/cart/cart.js:23]: You can use a utility CSS class here

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good. Clean code.
I would suggest creating utility CSS classes for margin and padding instead of using inline styles.

**Comment by GuptaEesh** [comment]
Got 2 reviews. 
>As far as utility classes are concerned I will add them to my CL first and then probably use it. As the CL review is on so I don't want the code to break by any mishap.

 Merging the branch now.

**Pull request merged**